leku/build.gradle

```diff
@@ -37,29 +37,33 @@ android {
 }
 
 dependencies {
-  compile fileTree(dir: 'libs', include: ['*.jar'])
+  implementation fileTree(dir: 'libs', include: ['*.jar'])
 
   def supportVersion = '26.1.0'
-  compile "com.android.support:appcompat-v7:$supportVersion"
-  compile "com.android.support:design:$supportVersion"
+  implementation "com.android.support:appcompat-v7:$supportVersion"
+  implementation "com.android.support:design:$supportVersion"
 
   def playServicesVersion = '11.4.2'
-  compile "com.google.android.gms:play-services-maps:$playServicesVersion"
-  compile "com.google.android.gms:play-services-location:$playServicesVersion"
+  implementation "com.google.android.gms:play-services-maps:$playServicesVersion"
+  implementation "com.google.android.gms:play-services-location:$playServicesVersion"
 
-  compile ('com.schibstedspain.android:android-reactive-location:0.10.1') {
+  implementation ('com.schibstedspain.android:android-reactive-location:0.10.1') {
     exclude group: 'com.google.android.gms'
   }
 
-  compile "io.reactivex:rxandroid:1.2.1"
-  compile 'io.reactivex:rxjava:1.3.2'
+  implementation "io.reactivex:rxandroid:1.2.1"
+  implementation 'io.reactivex:rxjava:1.3.3'
+  implementation 'io.reactivex.rxjava2:rxjava:2.1.6'
+  implementation 'io.reactivex.rxjava2:rxandroid:2.0.1'
```

Comment:
implementation 'io.reactivex:rxjava:1.3.3'
implementation 'io.reactivex.rxjava2:rxjava:2.1.6'
implementation 'io.reactivex.rxjava2:rxandroid:2.0.1'
All are necessary?

Reply:
Yes, all are necessary because I still need RxJava1 because we have a library that uses it and I cannot change it right now.